web/core/src/jsMain/kotlin/org/jetbrains/compose/web/elements/Base.kt

```diff
@@ -13,14 +13,13 @@ import org.w3c.dom.HTMLElement
 import org.w3c.dom.css.ElementCSSInlineStyle
 import org.w3c.dom.svg.SVGElement
 
-@OptIn(ComposeCompilerApi::class)
 @Composable
 @ExplicitGroupsComposable
-private inline fun <TScope, T> ComposeDomNode(
-    noinline factory: () -> T,
+private fun <TScope, T> ComposeDomNode(
```

Comment:
actually `attrsSkippableUpdate` can be inlined. 
Maybe it would be better?

`ComposeDomNode ` is used only from TagElement, so inlining won't hurt the bundle size, but can help with extra lambda instances allocations.

Reply:
not for this PR probably.